### PR TITLE
Validate

### PR DESCRIPTION
**Pull request by algo3228**
подправил трейн луп. Теперь есть отдельные функции на тренировку/валидацию. Кстати, нужно бы валидационный датасет сделать.

### CODE REVIEW

**Comment by algo3228** [comment]
за несколько эпох этот код даёт 60% accuracy. Кажется, что это маловато.

File: train.py [hunk 1]
```diff
@@ -1,35 +1,69 @@
...
+from torch.optim.lr_scheduler import StepLR
+import tqdm
 
 from dataset import BirdClefDataset, collate_fn
 from model import Network
+from utils import AverageMeter
 
 
-if __name__ == '__main__':
+def evaluate(model, loader):
```
Comment (TomHardy1138) [inline train.py:15]: .eval() для модели или декоратор @torch.no_grad()

**Review by TomHardy1138 — APPROVED** [review]

**Comment by algo3228** [comment]
добавил .eval(), метрику F1, train/test

**Comment by TomHardy1138** [comment]
Добавил MultiStepLR и сохранение весов после каждой эпохи
